src/test/java/org/broadinstitute/consent/http/resources/SupportResourceTest.java

```diff
@@ -0,0 +1,189 @@
+package org.broadinstitute.consent.http.resources;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+import com.google.api.client.http.HttpStatusCodes;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import jakarta.ws.rs.core.Response;
+import org.broadinstitute.consent.http.enumeration.SupportRequestType;
+import org.broadinstitute.consent.http.service.SupportRequestService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.zendesk.client.v2.model.Request;
+
+@ExtendWith(MockitoExtension.class)
+public class SupportResourceTest extends ResourceTest {
+
+  @Mock
+  private SupportRequestService supportRequestService;
+
+  private SupportResource supportResource;
+
+  @BeforeEach
+  public void setUp() {
+    supportResource = new SupportResource(supportRequestService);
+  }
+
+  @ParameterizedTest
+  @EnumSource(SupportRequestType.class)
+  void testPostRequestSuccess(SupportRequestType type) throws Exception {
+    String body = """
+        {
+          "name": "Test User",
+          "email": "test.user@example.com",
+          "subject": "Test Subject",
+          "description": "Test Description",
+          "type": "%s",
+          "url": "https://example.com",
+          "uploads": [
+            "token1",
+            "token2",
+          ]
+        }
+        """.formatted(type.name());
```

Comment:
`.name()` is redundant here
```suggestion
        """.formatted(type));
```